js/msp/MSPHelper.js

```diff
@@ -2409,6 +2427,10 @@ var mspHelper = (function (gui) {
         MSP.send_message(MSPCodes.MSPV2_INAV_MISC, false, false, callback);
     };
 
+    self.loadOutputMapping = function (callback) {
+        MSP.send_message(MSPCodes.MSPV2_INAV_OUTPUT_MAPPING, false, false, callback);
```

Comment:
@shellixyz please add a version check so call on older firmware wont cause a problem

Reply:
I'm not sure it is a good idea to add a check there. If we add a check and the function does nothing the data won't be loaded/refreshed in the OUTPUT_MAPPING var and the version will still need to be checked by the code using the OUTPUT_MAPPING data. For other messages the version is checked directly in the tab code itself.

Example from the receiver tab JS:
```
    if (semver.gte(CONFIG.flightControllerVersion, '1.8.1')) {
        loadChain.push(mspHelper.loadRateProfileData);
    } else {
        loadChain.push(mspHelper.loadRcTuningData);
    }
```

Reply:
but then, if 2 tabs will be asking for that frame, then 2 frames would have to include the logic to add only when FC offers it. I'm not sure what is happening on receiver tab is good too. adding a check on msphelper level makes implementation simpler in my opinon

Reply:
Right. I added a check `loadOutputMapping()` now returns false and clears `OUTPUT_MAPPING` if the firmware version is not compatible.